tests/models/gemma/test_modeling_gemma.py

```diff
@@ -531,12 +531,12 @@ def test_flash_attn_2_equivalence(self):
                 assert torch.allclose(logits_fa, logits, atol=3e-3)
 
 
-@require_torch_gpu
 @slow
-@require_read_token
+@require_torch_gpu
 class GemmaIntegrationTest(unittest.TestCase):
     input_text = ["Hello I am doing", "Hi today"]
 
+    @require_read_token
```

Comment:
Not sure I love this name. It sounds like this means `if no token, then test will skipped`. Any chance we can change the name (even in a follow up PR)?

Reply:
Yes let's do it in a follow up PR!